config_task_limits.go

```diff
@@ -36,6 +36,9 @@ type TaskLimitsConfig struct {
 	// MaxConcurrentLargeParserProjectTasks is the maximum number of tasks with >16MB parser projects that can be running at once.
 	MaxConcurrentLargeParserProjectTasks int `bson:"max_concurrent_large_parser_project_tasks" json:"max_concurrent_large_parser_project_tasks" yaml:"max_concurrent_large_parser_project_tasks"`
 
+	// MaxDegradedModeConcurrentLargeParserProjectTasks is the maximum number of tasks with >16MB parser projects that can be running at once during degraded mode.
+	MaxDegradedModeConcurrentLargeParserProjectTasks int `bson:"max_degraded_mode_concurrent_large_parser_project_tasks" json:"max_degraded_mode_concurrent_large_parser_project_tasks" yaml:"max_degraded_mode_concurrent_large_parser_project_tasks"`
+
 	// MaxDegradedModeParserProjectSize is the maximum parser project size during CPU degraded mode.
 	MaxDegradedModeParserProjectSize int `bson:"max_degraded_mode_parser_project_size" json:"max_degraded_mode_parser_project_size" yaml:"max_degraded_mode_parser_project_size"`
```

Comment:
Since this + the `MaxParserProjectSize` only applies to parser projects in S3 and not those in the DB, can we mention this? I noticed that `MaxGenerateTaskJSONSize` applies to all JSONs sent to generate.tasks (whether they end up in the DB or in S3), whereas the parser project size limit only applies to S3, so the difference in implementation decisions could be confusing.